sources/packages/api/src/app.students.module.ts

```diff
@@ -74,6 +75,8 @@ import { ApplicationControllerService } from "./route-controllers/application/ap
     GCNotifyService,
     GCNotifyActionsService,
     AssessmentControllerService,
+    StudentControllerService,
+    StudentFileService,
```

Comment:
This has been already imported in line 65 of the providers, can we please remove it here?